index.html

```diff
@@ -6,9 +6,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Classwork/Practice</title>
     <link rel="stylesheet" href="css/bootstrap.min.css">    
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-
+<div class="box container-fluid col-md-12 row yellow">[container-fluid]
+    <div class="container">
+        <div class="box header container col-md-12 row blue">[header]</div>
```

Comment:
Once you call in container as a parent, the rest of the divs don’t need it as long as their within the container. So in your case, I would remove all the class `containers` where you call in the `col-md-etc`